specs/lib/gltfPipelineSpec.js

```diff
@@ -120,7 +120,7 @@ describe('gltfPipeline', function() {
         readGltf(gltfEmbeddedPath, options, function (gltf) {
             var initialUri = gltf.buffers.CesiumTexturedBoxTest.uri;
             processJSON(gltf, options, function () {
-                var finalUri = gltf.buffers.CesiumTexturedBoxTest.uri;
+                var finalUri = gltf.buffers.uri;
```

Comment:
Oops, this was a naive change since `processFile` will now merge buffers, so `gltf.buffers.CesiumTexturedBoxTest` will no longer exist.

This could now expect the old buffer to be undefined and compare initialUri with `gltf.buffer_0.uri` (the only buffer that should be on the gltf), but that doesn't seem to be perfect test either. Thoughts?